helm/grafana/templates/grafana-dashboards.yaml

```diff
@@ -1,7486 +1,9 @@
 {{ define "grafana-dashboards.yaml.tpl" }}
-deployment-dashboard.json: |
```

Comment:
I think this got deleted by accident.

Reply:
I guess one of the scripts removed this line. I executed "make generate" in contrib/kube-prometheus and ran helm/hack/sync_kube_prometheus.yaml. 

Should "make generate" create grafana-dashboard.json?

Reply:
I'm assuming something there went wrong, it's probably best to just revert the changes in this file, but ultimately we should fix the scripts. (cc @gianrubio )

Reply:
I have reverted grafana-dashboards.yaml to it's previous state and pushed the new commit. Should be building now.